content/livesync/index.textile

```diff
@@ -1,48 +1,83 @@
+h3(#sequence-id). sequenceId
 
-When you update the data in your database, you record a change event transactionally in a dedicated "outbox table":https://docs.google.com/document/d/19BJ4oYC3ah4wFH0VS5Qv6qHe3qmRS_PpNLSytTOi9tc/edit#heading=h.r00ywdkts746. The outbox changes are detected by the Database Connector, which then publishes them as messages on channels. Using the "Models SDK":/livesync/models/models, your client applications subscribe to these change events and update their local state with the new data.
+The Models SDK utilizes a sequence ID to ensure accurate matching of the model's state with the corresponding point in the change events stream obtained from Ably. This sequence ID, retrieved from your backend during the @sync@ function call, enables the Models SDK to synchronize the model's state effectively during the initial load process.
 
-<aside data-type='note'>
-<p>The Database Connector is currently a "Docker container image":https://docs.docker.com/get-started/overview/#containers that can run on your own infrastructure. Please "reach out":https://docs.google.com/forms/d/e/1FAIpQLSd00n1uxgXWPGvMjKwMVL1UDhFKMeh3bSrP52j9AfXifoU-Pg/viewformif you have any feedback or prefer to use an Ably-hosted version of LiveSync.</p>
-</aside>
+h2(#use-cases). Use cases
 
-h2(#models-sdk). Models SDK
+LiveSync simplifies the synchronization of data between the database and frontend clients. LiveSync can benefit a wide range of applications including Customer Relationship Management (CRM) applications, customer support applications, productivity or task management applications, online auctions, collaborative form editing, ecommerce systems, chat conversations, multiplayer turn-based games (such as scrabble), and realtime newsfeeds.
 
-The "Models SDK":/livesync/models/models enables you to easily create live and observable data models in your frontend applications, ensuring they remain synchronized with your database's state in realtime. By subscribing to a model, you receive the changes made by other users, enabling the creation of a seamless, multiplayer application experience.
+Let's take the example of a CRM application to understand how LiveSync can enable reliable realtime data sync — In a CRM app, an individual customer details page can be represented as a single data model with its own channel. When a user updates the customer's information, optimistic updates can immediately reflect the changes in the UI while the confirmation goes to a roundtrip through the database. When confirmed, the original user's UI update can be confirmed and all other clients can also see that update without needing to refresh the page.
 
-The Models SDK works as follows:
+h3(#known-limitations). Known limitations
```

Comment:
LiveSync's current design might be limiting to data models based on graph data structures. As each data model is meant to work with a channel of it's own, any application changes that apply to multiple data models will need to be handled and published separately to the outbox table.

LiveSync might also be limiting to use-cases requiring complex conflict resolution logic and the "last write to the database wins" strategy isn't enough. Having said that, LiveSync can be easily coupled with the component locking feature in Spaces to make sure the individual UI components can be locked thus avoiding the need to resolve conflicts. For most use-cases like collaborative form building, component locking might actually a better user experience compared to co-editing capabilities.


Can we please replace this section with the above copy?